lago/plugins/vm.py

```diff
@@ -364,6 +365,10 @@ def metadata(self):
     def disks(self):
         return self._spec['disks'][:]
 
+    @property
+    def spec(self):
```

Comment:
Yes.
Any change to the spec will effect the structure of the env. IMO changes to the env should be done with specific object's methods (in this case the 'vm' object), and not directly to its data.